Source/Particles/ShapeFactors.H

```diff
@@ -23,170 +23,96 @@ struct Compute_shape_factor
 {
     template< typename T >
     AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
-    int operator()(T* const /*sx*/, T /*xint*/) const { return 0; }
-};
-
-/**
- *  Compute shape factor and return index of leftmost cell where
- *  particle writes.
- *  Specialization for order 0
- */
-template <>
-struct Compute_shape_factor< 0 >
-{
-    template< typename T >
-    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
-    int operator()(T* const sx, T xmid) const
+    int operator()(
+        [[maybe_unused]] T* const sx,
+        [[maybe_unused]] T xmid) const
```

Comment:
I would not put the `[[maybe_unused]]` in the interface but handle explicitly, to avoid errors:
```suggestion
        T* const sx,
        T xmid) const
```